### DISCUSSION TITLE
Not Added to My Wallet?

### DISCUSSION THREAD

**Discussion by RonaldHouw** [Chia-Network/chia-blockchain#6267] [Support]
I have question,
on the wallet-> history there is an entry of 0.75x (incoming) and it does not added to my wallet, and I have never withdrawn? I checked in Chia Explorer the status of 0.75xch is spent?
Why did it happen? Thanks

**Comment by BrandtH22** [comment]
@RonaldHouw , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
